### PR TITLE
SEO-175653-WPF-Pages-With-Redirects

### PR DESCRIPTION
**Pull request by berylouma**
Hello,

Worked on the links with redirects.

Regards,
Beryl.

### CODE REVIEW

File: wpf/Integer-TextBox/Appearance.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ This section deals with the appearance of `IntegerTextBox` control and contains
 
 ## Setting the Foreground
 
-The [IntegerTextBox](https://www.syncfusion.com/wpf-ui-controls/integer-textbox) control [Foreground](https://docs.microsoft.com/en-us/dotnet/api/system.windows.controls.control.foreground?view=netframework-4.8) can be modified based on the value of the control. The following are the foreground for `IntegerTextBox` control.
+The [IntegerTextBox](https://www.syncfusion.com/wpf-ui-controls/integer-textbox) control [Foreground](https://learn.microsoft.com/en-us/dotnet/api/system.windows.controls.control.foreground?view=netframework-4.8) can be modified based on the value of the control. The following are the foreground for `IntegerTextBox` control.
```
Comment (Yvone-Atieno) [inline wpf/Integer-TextBox/Appearance.md:16]: Update the correct link

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: wpf/Integer-TextBox/Appearance.md (hunk 1 above)
Comment (berylouma) [reply wpf/Integer-TextBox/Appearance.md]: Updated

**Review by Yvone-Atieno — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
Fine.

**Pull request merged**